var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -446,6 +455,11 @@ def filter_compilers(self):
             filter_file(env["CC"], cc_compiler, nrniv_makefile, **kwargs)
             filter_file(env["CXX"], cxx_compiler, nrniv_makefile, **kwargs)
 
+    @when("~cmake")
```

Comment:
Why the need now to do it? A code comment would be nice but to speed up just comment here.

Reply:
You're right. I will add one 👍